python/py-pytest-random-order/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-pytest-random-order
-version             1.0.4
+version             1.1.0
 revision            0
 
```

Comment:
Here and possibly elsewhere in this PR: if a Python port is "noarch" then please also set "platforms {darwin any}" - see an earlier email from @jmroot to the MacPorts devel-list and many of his recent commits to take care of that for unmaintained/openmaintainer ports.

Reply:
Please do **not** just blindly do this for all that are noarch though; only ones that install the same files on all OS versions. This will be the case for most, but it's quite possible for a noarch port to do something different based on the OS version. An easy way to check for python modules is if the wheels have `none-any` at the end of the filename.